### DISCUSSION TITLE
personal Creating Token

### DISCUSSION THREAD

**Discussion by sofiiberlin** [community/community#41552] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I try to create a personal token access and instead of seeing Configure SSO drop down menu I see Neve used message, how to change it? thank you
<img width="341" alt="Screen Shot 2022-12-12 at 8 38 18 P
<img width="1211" alt="Screen Shot 2022-12-10 at 7 00 08 PM" src="https://user-images.githubusercontent.com/112969387/207138268-dbbe850f-596f-4859-97a2-71036e2057ae.png">
M" src="https://user-images.githubusercontent.com/112969387/207138226-15a93997-1cd2-4ecc-a2c1-c16896e38f90.png">

**Comment by Elllkere** [comment]
same

**Comment by Abubakrce19** [comment]
If you are trying to create a personal access token on GitHub and you see a message saying "Never used" instead of the "Configure SSO" dropdown menu, it could be because you are not logged in to GitHub.

To create a personal access token, you will need to be logged in to your GitHub account. If you are not logged in, the "Configure SSO" dropdown menu will not be displayed.

To log in to your GitHub account, click on the "Sign in" button at the top right corner of the page. Then, enter your GitHub username and password to log in. Once you are logged in, you should be able to access the "Configure SSO" dropdown menu and create a personal access token.

If you are already logged in to your GitHub account and you are still seeing the "Never used" message instead of the "Configure SSO" dropdown menu, it could be because you do not have permission to create a personal access token. In this case, you may need to contact the owner of the repository or organization to request permission to create a personal access token.

Thanks

**Reply by Elllkere** [reply]
Im logged in and it still "Never used". But how can i not have permission to create a PAT on my personal account ? And i dont select any repo for PAT, i want to make it to clone private repo i have access to

**Comment by Abubakrce19** [comment]
If you are logged in and still seeing the message "Never used" when trying to create a personal access token (PAT) on your personal account, it could be that you don't have the necessary permissions to create a PAT.

To create a PAT, you will need to have the "Write" permission for the "Personal access tokens" scope. If you don't have this permission, you will not be able to create a PAT. This permission is usually granted to users who have administrative access to the account or organization.

If you are an administrator for the account or organization and you are still unable to create a PAT, it could be that your account has been restricted in some way. You may need to contact your system administrator or the person responsible for managing access to the account to have the restriction lifted.

Regarding cloning a private repository, you can use a PAT to authenticate to the repository if you have been granted access to it. To do this, you will need to create a PAT with the necessary permissions and then use it as the password when cloning the repository using a Git client. You can find more information on how to clone a repository using a PAT in the documentation for your Git client.

I hope this information helps!
